extras/craft/gulprev/twigextensions/GulpRevTwigExtension.php

```diff
@@ -30,12 +30,13 @@ public function gulpRev($file)
 
         // If the file is not defined in the asset manifest
         // just return the original string
-        $path = $file;
+        $path          = $file;
+        $manifest_path = $_SERVER['DOCUMENT_ROOT'] . '/rev-manifest.json';
```

Comment:
No, since `/craft` and `/public` are usually siblings.  It'd have to be `CRAFT_BASE_PATH . '../public/rev-menufest.json` and that might not be a correct relative path for everyone.